tools/azure-sdk-tools/packaging_tools/sdk_generator.py

```diff
@@ -7,7 +7,8 @@
 import shutil
 import re
 import os
-import pytoml as toml
+import tomli as toml
```

Comment:
Need to correct these to make them py311 safe. Won't pop in CI (which is an issue of itself) but adding a note here so I don't forget to fix both of those issues.